modules/gapi/test/common/gapi_stereo_tests_inl.hpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_TESTS_INL_HPP
+#define OPENCV_GAPI_STEREO_TESTS_INL_HPP
+
+#include <opencv2/calib3d.hpp>
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include "gapi_stereo_tests.hpp"
+
+namespace opencv_test {
+
+TEST_P(TestGAPIStereo, DisparityDepthTest)
+{
+    initMatsRandU(type, sz, dtype, true);
+
+    cv::gapi::StereoOutputFormat oF;
+    doDepths ? oF = cv::gapi::StereoOutputFormat::DEPTH :
+               oF = cv::gapi::StereoOutputFormat::DISPARITY;
+
+    // G-API
+    cv::GMat inL;
+    cv::GMat inR;
+    cv::GMat out = cv::gapi::stereo(inL, inR, oF, dtype);
+
+    auto kernels = cv::gapi::calib3d::cpu::kernels();
+
+    cv::GComputation(cv::GIn(inL, inR), cv::GOut(out))
+        .apply(cv::gin(in_mat1, in_mat2), cv::gout(out_mat_gapi),
+        cv::compile_args(kernels,
+                         cv::gapi::StereoInitParam {
+                                 numDisparities,
+                                 blockSize,
+                                 baseline,
+                                 focus
```

Comment:
Are these settings exclusive to `cv::StereoBM` only? If yes, than it's the CPU implementation detail and must go to the appropriate namespace.

Reply:
Done